tests/test_errata_data.py

```diff
@@ -267,6 +271,9 @@ def test_save_product_component_for_errata(
     build_list_url = f"{settings.ERRATA_TOOL_URL}/api/v1/erratum/{erratum_id}/builds_list.json"
     requests_mock.get(build_list_url, text=build_list)
     sb = SoftwareBuildFactory(build_id="1636922", build_type=BUILD_TYPE)
+    container_build = SoftwareBuildFactory(build_id="1628358", build_type=SoftwareBuild.Type.BREW)
+    container_build.meta_attr["nvr"] = "rh-dotnet31-container-3.1-18"
+    container_build.save()
```

Comment:
I hope you can set all these at creation time, and then you don't need .save():
```suggestion
    container_build = SoftwareBuildFactory(build_id="1628358", build_type=SoftwareBuild.Type.BREW, meta_attr={"nvr": "rh-dotnet31-container-3.1-18"})
```